EventFilter/CTPPSRawToDigi/src/RawToDigiConverter.cc

```diff
@@ -331,11 +332,10 @@ void RawToDigiConverter::run(const VFATFrameCollection &coll,
                                   eventInfoTmp);
           // calculate ids
           TotemTimingDetId detId(record.info->symbolicID.symbolicID);
-
           const TotemDAQMapping::TotemTimingPlaneChannelPair SWpair =
               mapping.getTimingChannel(totemSampicFrame.getHardwareId());
           // for FW Version > 0 plane and channel are encoded in the dataframe
-          if (totemSampicFrame.getFWVersion() < 0x30)  // Mapping not present in HW, read from SW for FW versions < 3.0
+          if (totemSampicFrame.getFWVersion() == 0)  // Mapping not present in HW, read from SW for FW versions < 3.0
```

Comment:
is this comment still consistent with the code?

Reply:
@jpata @cms-sw/orp-l2  given the urgency of this PR, can I propose to merge this as it is in the 2300 IB, submit a tiny PR that changes this afterwards, while change this in the backport PR? My worry is that if we change, then retriggering tests and waiting for the signatures again, we'll precious time. Unless you are going with doing a commit and merge w/o tests and re-signing? attn @ChrisMisan

Reply:
@clacaputo what do you think?

Reply:
@jpata @clacaputo please sign if you have no other concerns, or write them here otherwise.
We (either me or Qiang) can change the comment "< 3.0" into "= 0" right before merging, without the need to retrigger the tests and ask for additional signatures, if that's what is wanted.

Reply:
> We (either me or Qiang) can change the comment "< 3.0" into "= 0" r

Yes, that's what the backports have:
https://github.com/cms-sw/cmssw/pull/38232/files#diff-7d424a837075836cbf060d14db9612773ebc17eb454cbe7fda31cad26014b89fR338